curriculum/challenges/english/25-front-end-development/quiz-javascript-object-oriented-programming/66edd5267b52f8f617d8364f.md

```diff
@@ -17,439 +17,458 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+From the statements below, which one correctly defines OOP?
 
 #### --distractors--
 
-Placeholder distractor 1
+it is a programming model that is more concerned with functions.
 
 ---
 
-Placeholder distractor 2
+It is a programming model that is focused on the logic of the program.
 
 ---
 
-Placeholder distractor 3
+It is a programming model that applies to all programming problems to solve challenges.
 
 #### --answer--
 
-Placeholder answer
+It is a programming model concerned with data.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+The following are importances of OOP except ONE, which one?
 
 #### --distractors--
 
-Placeholder distractor 1
+Achieves security.
 
 ---
 
-Placeholder distractor 2
+Enhances code reusability.
 
 ---
 
-Placeholder distractor 3
+It makes a program flexible.
 
 #### --answer--
 
-Placeholder answer
+It makes the code simple.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following statements is true for the OOP?
 
 #### --distractors--
 
-Placeholder distractor 1
+Multiple instances of objects cannot exist.
 
 ---
 
-Placeholder distractor 2
+OOP can be applied in all cases.
 
 ---
 
-Placeholder distractor 3
+Procedural programs are more secure than OOP programs.
 
 #### --answer--
 
-Placeholder answer
+OOP programs are much larger and more complex to create.
```

Comment:
I'm not entirely sure this is accurate.

Reply:
They are relative to procedural programs.

Reply:
Trying to modify the answers, if not the questions.